operator/docs/operations.md

```diff
@@ -1,3 +1,45 @@
-# Pipeline Service operations
 
-## TODO
+### Introduction
+This document provides information on how Pipeline Service is operated and maintained, a SaaS for OpenShift Pipelines. It includes information on the project's working model, deployment modes, testing, security, and upgrade strategy of the operators. The document lists guidelines, which helps ensure a smooth operation of Pipeline Service.
+
+### Project Overview
+
+Pipeline Service is a SaaS (Software as a Service) platform for building and deploying continuous integration and continuous delivery (CI/CD) pipelines. It is designed to work with Kubernetes and OpenShift, and is primarily based on RedHat OpenShift Pipelines under the hood.
+
+More information on Pipeline Service is provided [here](../../README.md).
+ 
+Development Mode: To facilitate the usage of Pipeline Service, developers and SREs can easily try out the software by running it in Development mode on any OpenShift or Kubernetes cluster. Instructions and details on how to use the Development mode are available [here](../../developer).
+
+### Security
```

Comment:
Can we point to the location of where the tests are configured ?
Both for static and vulnerability scanning